cypress/e2e/supervision/documents/viewing-document.cy.js

```diff
@@ -0,0 +1,27 @@
+describe(
+  "Viewing a document in the documents tab for the client",
+  { tags: ["@supervision", "@deleting-document"] },
+  () => {
+
+    beforeEach(() => {
+      cy.loginAs("Allocations User");
+      cy.createClient();
+      cy.loginAs("Public API");
+      cy.uploadDocument();
+    })
+
+    it("allows document to be viewed with PDF viewer", () => {
+      cy.loginAs("System Admin");
+      cy.get("@client").then(({ id }) => {
+        cy.visit(`/supervision/#/clients/${id}`);
+      });
+
+      cy.get('#tab-container').contains('Documents').click();
+      cy.get('document-list-item').find('.tab-table').should('have.length', 1);
+      cy.get('.clear > .dotted-link').click();
+
+      cy.get('.document-viewer__iframe', {timeout: 30000}).should('be.visible');
+      cy.iframe('.document-viewer__iframe', {timeout: 30000}).find('.endOfContent').should('exist');
```

Comment:
Is there a reason for these timeouts? This is half the length of the default timeout, so I'm confused as to why we would want this to timeout sooner than default.

Reply:
No, I thought i was making it more failsafe as sometimes it takes a while to load but I forgot about the default one. I'll remove them.